src/index.html

```diff
@@ -13,6 +13,45 @@
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="product-card"
+      data-qa="card"
+    >
+      <img
+        src="./images/imac.jpeg"
+        alt="iMac img"
+        class="product-card__image"
+      />
+
+      <h3 class="product-card__name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h3>
+
+      <p class="product-card__code">Product code: 195434</p>
+
+      <div class="product-card__rating">
+        <div class="product-card__stars">
```

Comment:
```suggestion
        <div class="stars">
```
it should be separate bem block